roles/openshift_node/templates/node.yaml.v1.j2

```diff
@@ -6,7 +6,7 @@ dockerConfig:
   execHandlerName: ""
 imageConfig:
   format: {{ openshift.node.registry_url }}
-  latest: false
+  latest: true
```

Comment:
Earlier on in the beta we were also told it was required :smile:. I don't think we need this anymore. @miminar think we can revert these lines?